### PR TITLE
resolves issue #3

### PR DESCRIPTION
**Pull request by Blink29**

### CODE REVIEW

File: about.html [hunk 1]
```diff
@@ -22,7 +22,32 @@
 </head>
 <body>
   <!-- navbar -->
-  <main-header></main-header>
+  <main-header>
+    <nav class="navbar navbar-expand-md navbar-light bg-light">
```
Comment (Iamdivyak) [inline about.html:26]: delete line no. 26 to 49

File: index.html [hunk 2]
```diff
@@ -26,7 +26,32 @@
 
 <body>
   <!-- navbar -->
-  <main-header></main-header>
+  <main-header>
```
Comment (Iamdivyak) [inline index.html:29]: delete line no. 29 to 53

File: services.html [hunk 3]
```diff
@@ -22,7 +22,32 @@
 </head>
 <body>
   <!-- navbar -->
-  <main-header></main-header>
+  <main-header>
+    <nav class="navbar navbar-expand-md navbar-light bg-light">
```
Comment (Iamdivyak) [inline services.html:26]: 26 to 49 delete

File: projects.html [hunk 4]
```diff
@@ -22,7 +22,32 @@
 </head>
 <body>
   <!-- navbar -->
-  <main-header></main-header>
+  <main-header>
+    <nav class="navbar navbar-expand-md navbar-light bg-light">
```
Comment (Iamdivyak) [inline projects.html:26]: 26 to 49 delete

**Review by Iamdivyak — CHANGES_REQUESTED** [review]

**Comment by Blink29** [comment]
the active is rendered out it navbar.scss (bootstrap), its difficult to do it; either put nav in all pages like i did or use React for better

**Comment by Iamdivyak** [comment]
OK, @Blink29 thank you for your contribution. If you find the solution to this problem. feel free to add another issue to refactor the code. for now, I'm merging it. today, I also tried to solve this problem but couldn't

**Pull request merged**